js-es2015-react.md

```diff
@@ -0,0 +1,126 @@
+# Tooling
+
+For projects that use ES2015, we use:
+
+- eslint
+- eslint-plugin-react
+
+To lint our JavaScript files.
+
+# Ignores
+
+The `.eslintignore` file contains only `node_modules` and whatever project specific folders as required.
+
+# Styleguide
+
+General points:
+
+- Classes are PascalCased
+- Functions are camelCased
+- React Props and States are camelCased
+- Spaces, not tabs
+- Use propTypes where practicable
+- Promises are indented
+
+```
+Promise
+    .then(()=>{})
```

Comment:
according to eslint rule, this should have 2 spaces only and some spaces could be added, too :smirk: